File: Language/C#/Book/tmp/LINQ实战/LinqInAction/Chapter04.CSharp/Chapter04.Win/FormMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;


namespace LinqInAction.Chapter04.Win
{
  public partial class FormMain : Form
  {
    public FormMain()
    {
      InitializeComponent();
    }

    private void lnkFormStrings_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
    {
      new FormStrings().Show();
    }

    private void lnkFormBooks_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
    {
      new FormBooks().Show();
    }
  }
}

File: Language/C#/Book/tmp/LINQ实战/Chapter04.VB/Chapter04.Win/FormMain.vb
Public Class FormMain

  Private Sub lnkFormStrings_LinkClicked(ByVal sender As System.Object, ByVal e As System.Windows.Forms.LinkLabelLinkClickedEventArgs) Handles lnkFormStrings.LinkClicked
    Dim frm = New FormStrings()
    frm.Show()
  End Sub

  Private Sub lnkFormBooks_LinkClicked(ByVal sender As System.Object, ByVal e As System.Windows.Forms.LinkLabelLinkClickedEventArgs) Handles lnkFormBooks.LinkClicked
    Dim frm = New FormBooks()
    frm.Show()
  End Sub
End Class